### PR TITLE
Translates glossary.srt to spanish

### PR DESCRIPTION
**Pull request by maiskovich**
Translation of the file glossary.srt to spanish. Based on the english version in the folder en/appendices/glossary

### CODE REVIEW

File: es/appendices/glossary.rst [hunk 1]
```diff
@@ -1,44 +1,43 @@
...
-        and the class name is Toolbar.
+    Sintaxis de plugin
+        La Sintaxis de plugin se refiere al punto separador del nombre de la clase
+        indicación que la clase es parte de un plugin. Por ej.``DebugKit.Toolbar``.
+        El plugin es DebugKit y el nombre de la clase es Toolbar.
 
-    dot notation
-        Dot notation defines an array path, by separating nested levels with ``.``
-        For example::
+    Notación por puntos
```
Comment (markstory) [inline es/appendices/glossary.rst]: Glossaries are a bit unique in sphinx, as these terms can be used to generate links. You will also have to update all the places that ``:term:`<term name>``` is used to get the documentation to build correctly.  Another option is to not translate the term names and only translate the term definitions.

**Comment by maiskovich** [comment]
I made a new commit keeping the original names in english, adding a translation for the termin between brackets and translating the definition. Please let me know if this is correct or i should do it differently.

Sorry for the errors in the previous commit.

File: es/appendices/glossary.rst [hunk 2]
```diff
@@ -49,22 +52,22 @@ Glossary
...
+        Don't repeat yourself(No te repitas a ti mismo). Es un principio del desarrollo de software
+        apuntado a reducir la repetición de información de cualquier tipo.
+        En CakePHP DRY es usado para permitirte programar to código una vez, y re-usarlo en
+        distintas partes de tu aplicación.
 
 
 .. meta::
-    :title lang=es: Glossary
-    :keywords lang=es: html attributes,array class,array controller,glossary glossary,target blank,dot notation,routing configuration,forgery,replay,router,syntax,config,submissions
+:title lang=es: Glosario
```
Comment (antograssiot) [inline es/appendices/glossary.rst]: you need to rendent this line, the build is failling now

File: es/appendices/glossary.rst [hunk 3]
```diff
@@ -4,41 +4,44 @@ Glossary
...
-        They typically look like::
+        (Enrutamiento de array)
+        Un array de atributos que son pasados a :php:meth:`Router::url()`.
+        Usualmente se tipean de la siguiente manera::
 
             array('controller' => 'posts', 'action' => 'view', 5)
 
     HTML attributes
-        An array of key => values that are composed into HTML attributes. For example::
+    (Atributos HTML)
```
Comment (antograssiot) [inline es/appendices/glossary.rst]: you need more indentation here I think and some of the other below also. Your translation must not be aligned with the glossary element but with its detail

File: es/appendices/glossary.rst (hunk 2 above)
Comment (maiskovich) [inline es/appendices/glossary.rst]: What should i change there?
Should i keep the title in english? or its about indentation  in `:title lang=es: Glosario` ?
Thanks

Comment (antograssiot) [inline es/appendices/glossary.rst]: only the indentation before :title: 
The translation is fine.

**Comment by markstory** [comment]
Thanks :+1:

**Pull request merged**